README.md

```diff
@@ -104,11 +105,43 @@ Hakyll is going to build full URLs for them. Happy coding!
 Here is an example of the pull request that introduces the
 `hakyll-shortcut-links` library into the scope:
 
-* [Some Hakyll website](https://github.com/vrom911/vrom911.github.io/pull/32)
+- [Some Hakyll website](https://github.com/vrom911/vrom911.github.io/pull/32)
 
 As you can see by the diff the only actual change needed for that was replacing
 `pandocCompiler` with `allShortcutLinksCompiler`.
 
+#### Examples of the `@github` shortcut :
+
+- Link to a user :
+  ```
+  [kowainik](@github:kowainik)
+  [kowainik](https://github.com/kowainik)
+  ```
+  ```
+  [kowainik](@github(kowainik))
+  [kowainik](https://github.com/kowainik)
+  ```
```

Comment:
With the current docs, it's not immediately clear when shortcuts are used and how they are expanded. To me it seems that it would be better to have a markdown table like this one:

| Shortcut | Plain markdown |
| - | - |
| `[kowainik](@github)` | `[kowainik](https://github.com/kowainik)` |